README.md

```diff
@@ -9,6 +9,31 @@ This is a port of the TypesafeConfig library to C++.
 
 The library provides C++ support for the [HOCON configuration file format](https://github.com/typesafehub/config/blob/master/HOCON.md).
 
+## I Made Some Change
```

Comment:
This probably makes more sense as the commit message rather than part of the README.